NEWS.md

```diff
@@ -0,0 +1,32 @@
+
+
+<!--
+set cul   "cursorline
+cc=+1			"colorcolumn is 1 more than tw
+
+!pandoc % -t latex -V linkcolor:blue -V fontsize=12pt -V geometry:margin=0.5in -o ~/Downloads/print_and_delete/out.pdf
+
+-H header
+-V or --variable
+--pdf-engine=xelatex
+
+PANDOC EXAMPLES:
+https://learnbyexample.github.io/tutorial/ebook-generation/customizing-pandoc/
+
+MARKDOWN GUIDE:
+https://www.markdownguide.org/basic-syntax/
+
+-->
```

Comment:
Do you mind removing this comment?

Reply:
Comments:    Certainly can remove.   

I use neovim (not RStudio) and bottom line are control codes for page formatting.   Need to figure out work around.

Reply:
Given that this is just a comment I think it's fine to leave for the time being.